anchore_manager/cli/service.py

```diff
@@ -352,7 +339,9 @@ def start(services, auto_upgrade, anchore_module, skip_config_validate, skip_db_
 
                             in_sync = True
                         else:
-                            logger.warn("this version of anchore-engine requires the anchore DB version ({}) but we discovered anchore DB version ({}) in the running DB - it is safe to run the upgrade while seeing this message - will retry for {} more seconds.".format(str(code_versions['db_version']), str(db_versions['db_version']), str(max_timeout - int(time.time() - timer))))
+                            logger.warn(
+                                "this version of anchore-engine requires the anchore DB version ({}) but we discovered anchore DB version ({}) in the running DB - it is safe to run the upgrade while seeing this message - will retry for {} more seconds.".format(
```

Comment:
avoid `.format()` here

Reply:
This was just a change due to code reformatting. I'd like to review all of these later but not for this PR